test/end-to-end/cases-coverage/expected/stylesheet/xsl-variable-02-no-opt-coverage.html

```diff
@@ -0,0 +1,122 @@
+28: <span class="whitespace">      </span><span class="hit">&lt;/node&gt;</span>
+29: <span class="whitespace">      </span><span class="comment">&lt;!-- Local variable trivial with sequence constructor</span>
+30: <span class="comment">           xsl:sequence is needed to make it trivial, xsl:value-of causes inlined --&gt;</span>
+31: <span class="whitespace">      </span><span class="hit">&lt;node type="variable - trivial"&gt;</span>
+32: <span class="whitespace">        </span><span class="hit">&lt;xsl:variable name="trivial02"&gt;</span>
+33: <span class="whitespace">          </span><span class="hit">&lt;xsl:text&gt;</span><span class="hit">1</span><span class="hit">&lt;/xsl:text&gt;</span>
+34: <span class="whitespace">          </span><span class="hit">&lt;xsl:text&gt;</span><span class="hit">00</span><span class="hit">&lt;/xsl:text&gt;</span>
+35: <span class="whitespace">        </span><span class="hit">&lt;/xsl:variable&gt;</span>
+36: <span class="whitespace">        </span><span class="hit">&lt;xsl:sequence select="$trivial02" /&gt;</span>
+37: <span class="whitespace">      </span><span class="hit">&lt;/node&gt;</span>
+38: <span class="whitespace">      </span><span class="comment">&lt;!-- Local variable constant with select attribute --&gt;</span>
+39: <span class="whitespace">      </span><span class="hit">&lt;xsl:variable name="constant01" select="200" /&gt;</span><span class="whitespace">                          </span><span class="comment">&lt;!-- Expected miss (optim constant) --&gt;</span>
+40: <span class="whitespace">      </span><span class="hit">&lt;node type="variable - constant"&gt;</span>
+41: <span class="whitespace">        </span><span class="hit">&lt;xsl:value-of select="$constant01" /&gt;</span>
+42: <span class="whitespace">      </span><span class="hit">&lt;/node&gt;</span>
+43: <span class="whitespace">      </span><span class="comment">&lt;!-- Local variable constant with sequence constructor</span>
+44: <span class="comment">           Needed as attribute to make it constant --&gt;</span>
+45: <span class="whitespace">      </span><span class="hit">&lt;xsl:variable name="constant02" as="xs:string+"&gt;</span><span class="whitespace">                         </span><span class="comment">&lt;!-- Expected miss (optim constant) --&gt;</span>
+46: <span class="whitespace">        </span><span class="hit">&lt;xsl:text&gt;</span><span class="hit">2</span><span class="hit">&lt;/xsl:text&gt;</span><span class="whitespace">                                                 </span><span class="comment">&lt;!-- Expected miss (optim constant) --&gt;</span>
+47: <span class="whitespace">        </span><span class="missed">&lt;xsl:text&gt;</span><span class="missed">00</span><span class="missed">&lt;/xsl:text&gt;</span><span class="whitespace">                                                </span><span class="comment">&lt;!-- Expected miss (optim constant) --&gt;</span>
```

Comment:
I was surprised that with optimizations off, line 47 was missed. Line 46 was traced with `class="net.sf.saxon.expr.Literal"`, so why not line 47, too?